clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -253,10 +272,12 @@ void zoo() {
   [[intel::max_concurrency(2)]]
   [[intel::initiation_interval(2)]] for (int i = 0; i != 10; ++i)
       a[i] = 0;
-  [[intel::disable_loop_pipelining]]
+  [[intel::disable_loop_pipelining]] // expected-warning {{attribute 'intel::disable_loop_pipelining' is deprecated}} \
```

Comment:
I think we should replace use of [[intel::disable_loop_pipelining]] with [[intel::fpga_pipeline]] in this test instead of adding warnings. If we are deprecating  [[intel::disable_loop_pipelining]], it does not make sense to keep using it in our testing

Reply:
I have kept the deprecation warnings for few tests. I mostly replaced use of [[intel::disable_loop_pipelining]] with [[intel::fpga_pipeline]] in this test